.redhat-ci.yml

```diff
@@ -1,11 +1,7 @@
 host:
     distro: fedora/24/atomic
-packages:
-    - gcc
-    - python3-tox
-    - python3-virtualenv
 tests:
-    - tox
+    - docker run --rm -v /root:/root -v $(pwd):/code --workdir /code fedora:24 ./rhci.sh
```

Comment:
Updated to use ``:Z`` for the moment. Double checked the run as well and it worked :smile:.